---
title: Keynote Speakers
---

* <div>[Xihong Lin](https://www.hsph.harvard.edu/xihong-lin/), Harvard
  University<br />
  "Hypothesis Testing for Weak and Sparse Alternatives With Applications to Whole Genome Data"

    Massive genetic and genomic data generated using array and
    sequencing technology present many exciting opportunities as well
    as challenges in data analysis and result interpretation, e.g.,
    how to develop effective strategies for signal detection using
    massive genetic and genomic data when signals are weak and
    sparse. In this talk, I will discuss hypothesis testing for sparse
    alternatives in analysis of high-dimensional data motivated by
    gene, pathway/network based analysis in genome-wide association
    studies using arrays and sequencing data. I will focus on signal
    detection when signals are weak and sparse, which is the case in
    genetic and genomic association studies. I will discuss hypothesis
    testing for signal detection using variable selection based
    penalized likelihood based methods, the Generalized Higher
    Criticism (GHC) test, and the Generalized Berk-Jones test, and the
    robust omnibus test. I will discuss the challenges in statistical
    inference in the presence of both between-observation correlation
    and signal sparsity. The results are illustrated using data from
    genome-wide association studies and sequencing studies.

    -----

    ![Xihong Lin](lin.jpg){.imgfloat width=150px height=188px}
    Xihong Lin is Chair and Henry Pickering Walcott Professor of
    Biostatistics and Coordinating Director of Program in Quantitative
    Genomics of Harvard T. H. Chan School of Public Health, and
    Professor of Statistics of Harvard Faculty of Arts and Sciences
    She was Assistant, Associate, and Full professor of Biostatistics
    at the Universit of Michigan before she joined Harvard in 2005.
    She Received her BS in Applied Mathematics from Tsinghua
    University in 1989, MS in Statistics from University of Iowa in
    1991, and PhD in Biostatistics in 1994.
    
    Dr. Lin is a world-reknowned statistician for her contributions to
    mixed models, nonparametric and semiparametric regression, and
    statistical genetics and genomics. Her current interests lie in
    statistical genetics and genomics.  She is interested in
    development and application of statistical and computational
    methods for analysis of high-throughput genetic, genomic and
    ‘omics data in epidemiological, environmental and clinical
    sciences, especially genetic and epigenetic epidemiology, genes
    and environment, and medical genomics. Her current research
    projects include genome-wide array association studies, whole
    genome sequencing association studies, gene-environment
    interactions, and genome-wide DNA methylation studies, pathway and
    network analysis, and integrative genetics and genomics.  Her
    methodological work is currently supported by the MERIT award from
    the National Cancer Institute on "Statistical Methods for
    Correlated and High-Dimensional Biomedical Data,” and the PO1 from
    the National Cancer Institute on “Statistical Informatics in
    Cancer Research.” She is the contacting PI for the training grant
    entitled Joint training in biostatistics and computational
    biology.
    
    Dr. Lin received many distinguished honors, including the COPSS
    Presidents' Award in 2006, the Spiegelman award of the outstanding
    health statistician from the American Public Health Association in
    2002, and the MERIT Award from the National Cancer Institute
    (2007-2016).  She is a fellow of the American Statistical
    Association and the Institute of Mathematical Statistics in 2007.
    </div>

* <div>**[Dr. David Madigan](http://www.stat.columbia.edu/~madigan/)**, Columbia
  University<br />
  "Honest learning for the healthcare system: large-scale evidence
  from real-world data"<br />
  <small>(joint work with Martijn J. Schuemie, Patrick B. Ryan, George
  Hripcsak, and Marc A. Suchard)</small>

    In practice, our learning healthcare system relies primarily on
    observational studies generating one effect estimate at a time using
    customized study designs with unknown operating characteristics and
    publishing -- or not -- one estimate at a time. When we investigate
    the distribution of estimates that this process has produced, we see
    clear evidence of its shortcomings, including an over-abundance of
    estimates where the confidence interval does not include one
    (i.e. statistically significant effects) and strong indicators of
    publication bias. In essence, published observational research
    represents unabashed data fishing. We propose a standardized process
    for performing observational research that can be evaluated,
    calibrated and applied at scale to generate a more reliable and
    complete evidence base than previously possible, fostering a truly
    learning healthcare system. We demonstrate this new paradigm by
    generating evidence about all pairwise comparisons of treatments for
    depression for a relevant set of health outcomes using four large US
    insurance claims databases. In total, we estimate 17,718 hazard
    ratios, each using a comparative effectiveness study design and
    propensity score stratification on par with current state-of-the-art,
    albeit one-off, observational studies. Moreover, the process enables
    us to employ negative and positive controls to evaluate and calibrate
    estimates ensuring, for example, that the 95% confidence interval
    includes the true effect size approximately 95% of time. The result
    set consistently reflects current established knowledge where known,
    and its distribution shows no evidence of the faults of the current
    process. Doctors, regulators, and other medical decision makers can
    potentially improve patient-care by making well-informed decisions
    based on this evidence, and every treatment a patient receives becomes
    the basis for further evidence.

    -----

    ![David Madigan](madigan.jpg){.imgfloat width=150px height=200px}
    David Madigan is the Executive Vice-President for Arts & Sciences,
    Dean of the Faculty, and Professor of Statistics at Columbia
    University in the City of New York. He previously served as Chair of the
    Department of Statistics at Columbia University (2008-2013),
    Dean, Physical and Mathematical Sciences, Rutgers University (2005-2007),
    Director, Institute of Biostatistics, Rutgers University (2003-2004), and
    Professor, Department of Statistics, Rutgers University (2001-2007).
    He received his bachelor's degree in Mathematical Sciences
    (1984, First Class Honours, Gold Medal) and a
    Ph.D. in Statistics (1990), both from Trinity College Dublin.

    Dr. Madigan has over 160 publications in such areas as Bayesian statistics,
    text mining, Monte Carlo methods, pharmacovigilance and probabilistic
    graphical models. In recent years he has focused on statistical
    methodology for generating reliable evidence from large-scale
    healthcare data. From 2011 to 2014 he was a member of the FDA's
    Drug Safety and Risk Management Advisory Committee.

    Dr. Madigan is a fellow of the American Association of the Advancement
    of Science (AAAS), the Institute of Mathematical Statistics (IMS) and
    the American Statistical Association (ASA), and an elected member of
    the International Statistical Institute (ISI).  He served as
    Editor-in-Chief of Statistical Science (2008--2010) and Statistical
    Analysis and Data Mining, the ASA Data Science Journal (2013--2015).
    </div>

